modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java

```diff
@@ -54,6 +62,11 @@
     /** Arguments. */
     private BaselineArguments baselineArgs;
 
+    /**
+     * Use verbose mode for command output
+     */
+    private @NotNull boolean verbose = false;
```

Comment:
What about adding some description to command help output, that will clarify what verbose is doing in this particular case?
Please, pay attention, that if you'll decide to add such data you'll need to update few expected help outputs. For example: GridCommandHandlerClusterByClassTest_help.output